tests/test-cases/Group1-Docker-Commands/1-02-Docker-Pull.robot

```diff
@@ -136,7 +136,7 @@ Pull non-existent image
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull fakebadimage
     Log  ${output}
     Should Be Equal As Integers  ${rc}  1
-    Should contain  ${output}  image library/fakebadimage not found
+    Should contain  ${output}  pull access denied
```

Comment:
Same question again - I'd expect either TagNotFound or ImageNotFound errors for an image that straight up doesn't exist. What I get from trying my local docker is a message that expresses either/or:
```
$ docker pull fakebadimage
Using default tag: latest
Error response from daemon: repository fakebadimage not found: does not exist or no pull access
```